internal/cmd/base/dev.go

```diff
@@ -39,7 +40,11 @@ func (b *Server) CreateDevDatabase(ctx context.Context, opt ...Option) error {
 
 	switch b.DatabaseUrl {
```

Comment:
I realize that there are not existing unit tests for this receiever func... but would it be possible to add one that tests the new option's behavior?

Reply:
I can look into this. This code path is exercised by test that use the `base.CreateDevDatabase`, but having a dedicated test for it would be a good idea if it isn't too complicated to add (given that there isn't an existing test around this).